integration-tests/nft/test_gallery.exp

```diff
@@ -95,3 +95,21 @@ task 15 'run'. lines 236-250:
     "Keep": "Executed"
   }
 }
+
+task 16 'run'. lines 252-269:
+{
+  "gas_used": 519398,
+  "status": {
+    "Keep": {
+      "MoveAbort": [
+        {
+          "Module": {
+            "address": "0x00000000000000000000000000000001",
+            "name": "Vector"
+          }
+        },
+        3
+      ]
+    }
```

Comment:
expect status should be Executed, we need to figure out why to abort at Vecotor module.